src/components/vocab/accordion.js

```diff
@@ -20,6 +20,20 @@ class VocabAccordion extends Accordion {
     return <IconBook16/>
   }
 
+  get toolbar() {
```

Comment:
@johanneskrtek correct me if I'm wrong, but I think we'll want to put the button into the accordion header with the delete icon. Both icons should not be visible at all for protected vocabularies.

Reply:
for context: <img width="578" alt="screen shot 2017-11-01 at 11 38 39" src="https://user-images.githubusercontent.com/293400/32271251-4c766518-bef9-11e7-975f-190fa2712af8.png">

Reply:
Yep, icon looks better there 
<img width="570" alt="screen shot 2017-11-01 at 12 03 55" src="https://user-images.githubusercontent.com/293400/32272049-c21385a0-befc-11e7-80f9-f900f211771e.png">